src/Workleap.DomainEventPropagation.Publishing/TracingPublishingDomainEventBehavior.cs

```diff
@@ -8,13 +8,7 @@ internal sealed class TracingPublishingDomainEventBehavior : IPublishingDomainEv
 {
     public async Task HandleAsync(DomainEventWrapperCollection domainEventWrappers, DomainEventsHandlerDelegate next, CancellationToken cancellationToken)
     {
-        if (domainEventWrappers.DomainSchema == EventSchema.CloudEvent)
-        {
-            await next(domainEventWrappers, cancellationToken).ConfigureAwait(false);
-            return;
-        }
-
-        var activityName = TracingHelper.GetEventGridEventsPublisherActivityName(domainEventWrappers.DomainEventName);
+        var activityName = GetPublishingActivityName(domainEventWrappers);
```

Comment:
When publishing an event, we can parameterize the activity name based on the event schema and also when receiving the event type.
Note: we only support EventGridEvent for custom topics and CloudEvent for Namespace topics.